### PR TITLE
added missing variables

### PR DESCRIPTION
**Pull request by LucasSilbernagel**
Adds missing variables: `github_link` and `gallery_dirs`

@josh146 I don't really know what I'm doing with this, let me know if I missed anything or if there's anything else I can do to help.

### CODE REVIEW

**Comment by Mandrenkov** [comment]
Have you tried building a local version of the e.g. PennyLane documentation using this version of the XST and confirmed that the expected behaviour is happening now, @LucasSilbernagel?

File: README.rst [hunk 1]
```diff
@@ -61,6 +61,10 @@ Once installed, simply add or modify the following variables of your Sphinx
 
         "navbar_home_link": "https://example.com",
 
+        "github_link": "",
+        
+        "gallery_dirs": [],
+
```
Comment (josh146) [inline README.rst:67]:
```suggestion
        "github_link": "",
```

Comment (josh146) [reply README.rst:67]: until we know what this does, I would remove it 😆

File: xanadu_sphinx_theme/theme.conf [hunk 2]
```diff
@@ -53,6 +53,11 @@ google_analytics_tracking_id =
 # List of extra copyright notices to place in the footer.
 extra_copyrights =
 
+# Link that is opened when View on GitHub is clicked on a demo
```
Comment (josh146) [inline xanadu_sphinx_theme/theme.conf:56]:
```suggestion
# GitHub organization and repository name of associated GitHub repo.
# Used to ensure that 'View on GitHub' buttons work.
```

**Review by josh146 — CHANGES_REQUESTED** [review]
Thanks @LucasSilbernagel! just two minor changes.

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,17 @@
-## Release 0.4.0 (development release)
+## Release 0.3.2
+
+### Bug fixes
+
+* Added missing variables: `github_link` and `gallery_dirs` to address the View on GitHub link not working for demos.
```
Comment (josh146) [inline .github/CHANGELOG.md:5]:
```suggestion
* Added missing variables: `github_repo` and `gallery_dirs` to address the View on GitHub link not working for demos.
```

File: README.rst [hunk 4]
```diff
@@ -61,6 +61,10 @@ Once installed, simply add or modify the following variables of your Sphinx
 
         "navbar_home_link": "https://example.com",
 
+        "github_link": "",
```
Comment (josh146) [inline README.rst:64]:
```suggestion
        "github_repo": "",
```

File: README.rst [hunk 5]
```diff
@@ -119,6 +123,11 @@ the ``html_theme_options`` dictionary in your ``conf.py`` file.
 
 ``google_analytics_tracking_id``
     Google Analytics tracking ID to enable website analytics.
+    
+``github_link``
+    The GitHub organization and repository associated with the documentation. E.g.,
+    for a GitHub repository https://github.com/Organization/repo, this should be
+    ``"github_link": "Organization/repo"``.
```
Comment (josh146) [inline README.rst:130]:
```suggestion
``github_repo``
    The GitHub organization and repository associated with the documentation. E.g.,
    for a GitHub repository https://github.com/Organization/repo, this should be
    ``"github_repo": "Organization/repo"``.
```

File: xanadu_sphinx_theme/theme.conf [hunk 6]
```diff
@@ -53,6 +53,11 @@ google_analytics_tracking_id =
...
+github_link =
```
Comment (josh146) [inline xanadu_sphinx_theme/theme.conf:57]:
```suggestion
github_repo =
```

**Review by josh146 — CHANGES_REQUESTED** [review]
@LucasSilbernagel I just noticed that this variable is called `github_repo` here, not `github_link`:

https://github.com/XanaduAI/xanadu-sphinx-theme/blob/0ce956830ee6c3a9f69964164e352dc0f72b61ae/xanadu_sphinx_theme/layout.html#L298

So a few suggested changes to correct this.

**Comment by LucasSilbernagel** [comment]
> Have you tried building a local version of the e.g. PennyLane documentation using this version of the XST and confirmed that the expected behaviour is happening now, @LucasSilbernagel?

@Mandrenkov @josh146 are there instructions somewhere on how to do that?

**Comment by josh146** [comment]
@LucasSilbernagel if you install this repository locally using pip (`pip install -e .`), then `make html-norun` on the QML website will use this branch.

**Comment by LucasSilbernagel** [comment]
> @LucasSilbernagel if you install this repository locally using pip (`pip install -e .`), then `make html-norun` on the QML website will use this branch.

Thanks @josh146 ! I think something still isn't right, the View on GitHub links still don't work.

**Comment by josh146** [comment]
@LucasSilbernagel have you updated the `qml/conf.py` file to specify `'github_repo':'PennyLaneAI/qml'`?

**Review by josh146 — APPROVED** [review]

**Comment by LucasSilbernagel** [comment]
> @LucasSilbernagel have you updated the `qml/conf.py` file to specify `'github_repo':'PennyLaneAI/qml'`?

@josh146 Ah okay, that worked! Do I need to open QML PR with that change?

**Comment by josh146** [comment]
@LucasSilbernagel yes please 🙂 Note that it won't work until we make a new release of XST. Do you mind updating the `xst/_version.py` file to reflect `0.3.2`, so that we can make a release?

**Comment by LucasSilbernagel** [comment]
@josh146 version updated for release. Is there anything else we have to do here, or can this just be merged?

**Comment by josh146** [comment]
Should be good to be merged @LucasSilbernagel :)

**Pull request merged**